src/v2/Components/Authentication/Desktop/SignUpForm.tsx

```diff
@@ -112,6 +114,18 @@ export class SignUpForm extends Component<FormProps, SignUpFormState> {
                 onChange={handleChange}
                 onBlur={handleBlur}
               />
+              <EmailSubscriptionCheckbox
+                error={
+                  touched.agreed_to_receive_emails &&
+                  errors.agreed_to_receive_emails
+                }
```

Comment:
```suggestion
```

I _think_ we can remove this entirely - there's no error state for this checkbox, you can check it or not, both are valid.

Reply:
Removed these.